.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       - image: circleci/python:3.8
     steps:
       - checkout
-      - run: pip install flake8
+      - run: pip install flake8 flake8-docstrings
```

Comment:
It is an interesting idea and I don't see the harm in it.

Would be good to add to the pre-commit config as well